news/2025/2025-03-25-the-followpoint-special-behind-the-banhammer.md

```diff
@@ -0,0 +1,292 @@
+
+If you are facing any issues with your account, please consult the article above and direct your concerns to one of the outlined inboxes.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+And lastly, what is your favorite GMT out of context quote?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7944724" style="background-image: url('/wiki/shared/avatars/0x84f.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/7944724">0x84f</a></p>
+
+"Hey, the."
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>
+
+"Hey, the."
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>
+
+"Hey, the."
+
+</div>
+
+---
+
+I'd like to thank all Global Moderation Team members for everything they do for osu! and their efforts to make chats a nicer place! I know that sometimes it doesn't seem like it, but they are really trying their best, so please don't let some bad apples from the past make you think all GMTs are like this, I know some of them myself and they are amazing people. <3
+And of course thank you 0x84f, Kobold84 and Venix for agreeing to this interview and answering my questions, I'm really glad I was given this opportunity to show a wider audience a bit more of what the GMT really is!
+
+Now I'll say a few words about The Followpoint itself. After our [last interview](https://osu.ppy.sh/home/news/2024-12-05-the-followpoint-hivie-the-man-of-many-things) things slowed down due to a lot of things happening in our lifes, but now as we recover — things should get back to normal. Anyway, I really hope you enjoyed this special interview, and thank you for reading it!
+
+Right now we cannot promise the next interview will come out on schedule like before, but please look forward to a new interview coming out hopefully soon enough, and make sure to keep an eye on the news feed! You may also follow us [on Bluesky](https://bsky.app/profile/thefollowpoint.megamix.dev) or [on Twitter](https://twitter.com/The_Followpoint) to see more updates and question submissions for future interviews.
```

Comment:
may want to consider not promoting twitter/bsky on single every post in future iterations (we don't do this kind of thing elsewhere)

Reply:
Fair, in this newspost I think it makes sense considering we had a big pause, but in the future yeah, I agree.